test/src/test/java/hudson/model/DirectoryBrowserSupportTest.java

```diff
@@ -27,6 +27,7 @@
 import com.gargoylesoftware.htmlunit.UnexpectedPage;
 import com.gargoylesoftware.htmlunit.html.HtmlPage;
 import com.gargoylesoftware.htmlunit.util.NameValuePair;
+import com.sun.management.UnixOperatingSystemMXBean;
```

Comment:
This type is exported and is safe to use 

https://docs.oracle.com/javase/8/docs/jre/api/management/extension/com/sun/management/UnixOperatingSystemMXBean.html

https://docs.oracle.com/javase/8/docs/jdk/api/javac/tree/jdk/Exported.html